cmd/common.go

```diff
@@ -74,7 +81,11 @@ func AddCommonFlags(cmd *cobra.Command, cfg *CommonConfig) {
 	}
 }
 
-func CheckCommonConfig(cfg *CommonConfig) error {
+func CheckCommonConfig(cmd *cobra.Command, cfg *CommonConfig) error {
+	if cmd.PersistentFlags().Changed("store-key") {
```

Comment:
I'll also add a check before this line that will return an error if both are provided (to avoid any ambiguity):

```
	if cmd.PersistentFlags().Changed("store-key") && cmd.PersistentFlags().Changed("store-key-file") {
		return fmt.Errorf("only one of --store-key and --store-key-file should be provided")
	}
```